src/static/riot/competitions/form/_phases.tag

```diff
@@ -248,13 +248,26 @@
                     setTimeout(function () {
                         self.simple_markdown_editor.codemirror.refresh()
                     }.bind(self.simple_markdown_editor), 10)
+                },
+                onHidden: () => {
+                    self.clear_form()
```

Comment:
very cool, we haven't been using the fat arrow syntax much, do you like it? Think we should make that the CKC design standard?

Reply:
Yea I definitely prefer fat arrow syntax for sure.